db/migrate/20210204171358_update_invalid_telephone_numbers.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateInvalidTelephoneNumbers < ActiveRecord::Migration[6.1]
+  def change
+    %w[primary_telephone
+      secondary_telephone].each do |phone_field|
+        Patient.where(purged: false).where.not("#{phone_field} REGEXP ?", '^\\+1(0|[2-9])[0-9]{9}$').each do |patient|
+          if patient[phone_field].sub(/^\+/, '').length == 10 && !patient.address_state.nil?
```

Comment:
Why do we check if `address_state` is `nil` here? I saw in one of your comments about checking for USA addresses but it doesn't look like we're doing that?

Reply:
My thinking was that we only wanted to add a `1` to the beginning of a phone number that was a USA address, and I was assuming that the presence of `address_state` was sufficient to indicate if an address uses the `+1` country code or not. Because if `address_state` is specified, then it must be one of the US states or territories. Is that assumption wrong?

Reply:
Ah I see. I think that's probably sufficient. I'd like to run a version of this script on prod that gets us the count of these first, before we actually modify anything.